migrate/scripts/gauge-name.js

```diff
@@ -0,0 +1,68 @@
+const qs = require('querystring');
+
+const urltoGaugeName = url => {
+  if (!url) return url;
```

Comment:
We can safely assume that `url` will be valid given the filtering we do elsewhere in the script.